js/src/redux/providers/signerReducer.js

```diff
@@ -90,7 +90,7 @@ export default handleActions({
   signerSuccessRejectRequest (state, action) {
     const { id } = action.payload;
     const rejected = Object.assign(
-      state.pending.find(p => p.id === id),
+      state.pending.find(p => p.id === id) || { id },
```

Comment:
Same as the previous fix (now for reject), I missed this case the first time around. (Let's hope it is indeed dead now)